src/models/data.h

```diff
@@ -468,6 +468,18 @@ struct TopDownResults
     static TopDownResults fromBottomUp(const Data::BottomUpResults& bottomUpData);
 };
 
+struct FrequencyData
+{
+    quint64 time;
+    qreal cost;
+};
+
+struct FrequencyResults
+{
+    QHash<int, QVector<FrequencyData>> data;
```

Comment:
what does this map, is this sparse or dense? if it's dense, then these should be plain vectors.

can you add a comment at least to indicate what the `int` is here?

Reply:
I just see that you are using this in a dense manner in the model, so these two things should not be hashes but vectors instead